.github/workflows/pr.yaml

```diff
@@ -20,11 +20,11 @@ jobs:
     steps:
       - uses: mheap/github-action-required-labels@v1
         with:
-          mode: exactly
+          mode: minimum
           count: 1
-          labels: "kind/feature, kind/fix, kind/tests, breaking-backwards-compitabillity, chore/ci, chore/documentation"
+          labels: "feature, bugfix, tests, breaking-backwards-compitabillity, ci, documentation"
```

Comment:
typo in compatibility, verify that's it's correct also in label? (I think I fixed it, not sure...)

Reply:
I'm just deleting it